decidim-proposals/config/locales/en.yml

```diff
@@ -57,6 +54,9 @@ en:
             votes_blocked: Votes blocked
             votes_enabled: Votes enabled
             votes_hidden: Votes hidden (if votes are enabled, checking this will hide the number of votes)
+    content_renderers:
+      proposal_renderer:
+        proposal_not_found: the proposal that was referenced here with id %{proposal_id} but doesn't exist anymore
```

Comment:
I don't understand this sentence, it's gramatically weird, but I don't understand the use case. For what I understand:

1. I leave a comment with a reference to a proposal: `Hey, check the proposal #123 which is related to yours!`
1. For some reason, the proposal I linked to gets removed
1. Comment now reads: `Hey, check the proposal the proposal that was referenced here with id 123 but doesn't exist anymore which is related to yours!`

Is that so? If so, I think the behaviour is weird. If proposal is not found, can we render the original link/proposal ID or similar? Something that is not a sentence? 😕 

Although you might want to ask product for this 🤷‍♂️

Reply:
A proposal can't be removed, just withdrawn. When this happens (2 and 3) I would say that we should leave the link in the comment but we shouldn't show any other automatic message or just show something like: "The proposal referenced has been withdrawn", but I understand the rendering problem for this.

Reply:
@arnaumonty @decidim/product I'm removing the code that controls this and let the processor render the link as usual.

Reply:
@arnaumonty @decidim/product reviewing what whe have commented I think (please @decidim/lot-core confirm) that the url for an unexisting proposal can not be build only from an ID as the url contains its feature and space id and we can not deduce them from anywhere.

@decidim/product Probably this was the cause because I added this "feature". So, as it can not happen that a Proposal disappears from DB, can we ignore this corner case?

Reply:
@tramuntanal yes you can, use `ResourceLocatorPresenter.new(proposal).path` or `.url` to retrieve the URL of the proposal. It takes care of finding the feature ID and the space type and ID, and builds the correct URL.

Reply:
@mrcasals no, no, we're talking about the case that we have a proposal global id, but this proposal does not exist in the database.

Reply:
@decidim/product I took the risk of being proactive and removed the spec because as this can not happen, there is not need to control it.